### PR TITLE
[DT-146] Added new strict ruleset

### PR DESCRIPTION
**Pull request by pushokwhite**
#### Overview

- Ticket: https://spryker.atlassian.net/browse/DT-146

###### Optional

- Core PR: https://github.com/spryker/spryker/pull/4632

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -97,6 +100,17 @@ This can be configured in the Development bundle config:
...
         return '/path/to/your/ruleset.xml';
     }
+    
+    /**
+     * Either a relative or full path to the stricter ruleset.xml or a name of an installed
+     * standard (see `phpcs -i` for a list of available ones).
+     *
+     * @return string
+     */
+    public function getCodingStandardStrict()
```
Comment (dereuromark) [inline README.md]: Do we need this?

Comment (pushokwhite) [reply README.md]: As we discussed projects should have only one CS level, so for the projects it is optional. But if we want to use `console c:s:s -l 2` - we should implement this.

Comment (dereuromark) [reply README.md]: hmm i see your point.

Comment (dereuromark) [reply README.md]: Project always only uses one standard, so we dont need this.

Comment (pushokwhite) [reply README.md]: Okay, so I will remove those lines.

**Pull request merged**